src/main/java/de/slimecloud/slimeball/features/staff/absence/AbsenceCommand.java

```diff
@@ -0,0 +1,97 @@
+package de.slimecloud.slimeball.features.staff.absence;
+
+import de.mineking.discordutils.commands.ApplicationCommand;
+import de.mineking.discordutils.commands.ApplicationCommandMethod;
+import de.mineking.discordutils.commands.Setup;
+import de.mineking.discordutils.commands.option.Option;
+import de.slimecloud.slimeball.config.GuildConfig;
+import de.slimecloud.slimeball.main.CommandPermission;
+import de.slimecloud.slimeball.main.Main;
+import de.slimecloud.slimeball.main.SlimeBot;
+import lombok.RequiredArgsConstructor;
+import net.dv8tion.jda.api.EmbedBuilder;
+import net.dv8tion.jda.api.events.interaction.command.SlashCommandInteractionEvent;
+import net.dv8tion.jda.api.utils.TimeFormat;
+import org.jetbrains.annotations.NotNull;
+
+import java.time.*;
+
+@RequiredArgsConstructor
+@ApplicationCommand(name = "absence", description = "Abwesenheit an/abmelden")
+public class AbsenceCommand {
+	private final SlimeBot bot;
+	public final CommandPermission permission = CommandPermission.ROLE_MANAGE;
+
+	@Setup
+	public static void setup(@NotNull SlimeBot bot) {
+		bot.scheduleDaily(12, new Absence(bot).check());
+	}
+
+	@ApplicationCommandMethod
+	public void performCommand(@NotNull SlashCommandInteractionEvent event,
+	                           @Option(description = "Bist du Abwesend") boolean absent,
+	                           @Option(description = "Der Grund warum du weg bist", required = false) String reason,
+	                           @Option(description = "Der Tag, an dem du zurück bist", required = false, minValue = 1, maxValue = 31) Integer day,
```

Comment:
This should probably be a string input in the format `dd.MM.yyyy`. Having 3 separate options is a little confusing I think

Reply:
i looked at the Birthday feature how we solved it there